### PR TITLE
[ONLINE-749] migrate repository to jitpack 

### PR DESCRIPTION
**Pull request by swVisenze**

### CODE REVIEW

File: cameraDemo/src/main/java/com/visenze/visearch/camerademo/views/adapter/StrechImageAdapter.java [hunk 1]
```diff
@@ -78,8 +78,10 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
 
-        int score = (int)(imageList.get(position).getScore() * 100);
-        viewHolder.scoreView.setText("Similarity " + String.valueOf(score) + "%");
+        Float score = imageList.get(position).getScore();
+        if (score != null) {
+            viewHolder.scoreView.setText("Similarity " + String.valueOf((int)(score * 100)) + "%");
+        }
 
         return imageLayout;
     }
```
Comment (thehung111) [inline cameraDemo/src/main/java/com/visenze/visearch/camerademo/views/adapter/StrechImageAdapter.java:87]: @swVisenze how come there are Apache HTTP client and visearch android JAR under dependency folder? can you remove ?

File: visearch-android/build.gradle [hunk 2]
```diff
@@ -40,44 +24,20 @@ android {
...
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
+
 dependencies {
-    compile 'com.visenze.datatracking:tracking:0.1.9'
-    compile 'com.squareup.retrofit2:retrofit:2.3.0'
-    compile 'com.squareup.retrofit2:converter-gson:2.3.0'
+    implementation 'com.visenze.datatracking:tracking:0.1.9'
```
Comment (thehung111) [inline visearch-android/build.gradle:36]: i think previously Yu Lu revered to compile due to some issues reported by Daniel's customers. Could you double check whether changing to implementation is OK in a demo app?

File: visearch-android/src/main/java/com/visenze/visearch/android/model/FacetRange.java [hunk 3]
```diff
@@ -4,24 +4,24 @@
 
 public class FacetRange {
     @SerializedName("min")
-    private Number min;
+    private String min;
```
Comment (thehung111) [inline visearch-android/src/main/java/com/visenze/visearch/android/model/FacetRange.java:7]: how come this is String? should be number

File: visearch-android/build.gradle (hunk 2 above)
Comment (thehung111) [reply visearch-android/build.gradle:36]: https://github.com/visenze/visearch-sdk-android/commit/f892f4daf1ed71f0fa4248d61a5e722d5af09f81#diff-4035af35a3c9e4da646562bfbf0cdd6ae1c41a6c4cd079b5e9f6dcc472b4e8a0 this is the commit where Yu Lu reverted

File: cameraDemo/src/main/java/com/visenze/visearch/camerademo/views/adapter/StrechImageAdapter.java (hunk 1 above)
Comment (swVisenze) [reply cameraDemo/src/main/java/com/visenze/visearch/camerademo/views/adapter/StrechImageAdapter.java:87]: this pr is to remove the jar files under dependency folders. all jar files in depencency folder is removed .

File: visearch-android/build.gradle (hunk 2 above)
Comment (swVisenze) [reply visearch-android/build.gradle:36]: i believe use compile is the limitation of the bintray repository. as in this pr it is moved to jitpack. the github and jitpack do not have that limitation. i have tried on my side. @yulu

File: cameraDemo/src/main/java/com/visenze/visearch/camerademo/views/adapter/StrechImageAdapter.java (hunk 1 above)
Comment (thehung111) [reply cameraDemo/src/main/java/com/visenze/visearch/camerademo/views/adapter/StrechImageAdapter.java:87]: oh I see, thanks, i thought it is added

File: visearch-android/src/main/java/com/visenze/visearch/android/model/FacetRange.java (hunk 3 above)
Comment (swVisenze) [reply visearch-android/src/main/java/com/visenze/visearch/android/model/FacetRange.java:7]: this is changed by shannon. he tested number  does not work.

File: visearch-android/build.gradle (hunk 2 above)
Comment (thehung111) [reply visearch-android/build.gradle:36]: ok if you have tested in an app, then it is fine

File: visearch-android/src/main/java/com/visenze/visearch/android/model/FacetRange.java (hunk 3 above)
Comment (thehung111) [reply visearch-android/src/main/java/com/visenze/visearch/android/model/FacetRange.java:7]: ok

**Review by thehung111 — APPROVED** [review]

**Pull request merged**